change/@ni-nimble-components-f53bdc20-40c9-40ed-9375-2b99fccc4bfd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "changes",
```

Comment:
Typically, we'd like the `comment` field to have a concise, but explanatory description of what these changes are providing.